torch/_dynamo/guards.py

```diff
@@ -472,14 +472,18 @@ def SHAPE_ENV(self, guard: Guard):
         # NB: self.output_graph can be None in the debug_nops tests
         fs = output_graph.tracked_fakes
         constraint_inputs = [a.constraint_dims for a in fs]
-        guards = output_graph.shape_env.produce_guards(
+        (
+            guards,
+            shape_constraints,
```

Comment:
These are only input shape constraints, not include inline shape constraints due to unbacked symint right?

Reply:
Sure